mantidproject/mantid
# Changed from mac line endings in Bilby instrument definition

this was causing the file to download every startup as our checksumhelper did not correctly understand old mac line endings


Just changed line endings in the file from the old mac endings

**To test:**

1. Check the file does not have old mac <cr> style line endings, and there is no there is no other change to the file.


No issue

*Does not need to be in the release notes.*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
